Update sequence.md

ellenevans opened the pull request:
The document to which this links no longer exists. I'm guessing the equivalent is "CloudSchedulerSource" but the code is possibly not the same.  So I've removed the link to stop the error.

<!-- General PR guidelines:

Most PRs should be opened against the master branch.

If the change should also be in the most recent release, add the
corresponding "cherrypick-0.X" label; for example, "cherrypick-0.12", to the
original PR. Best practice is to open a PR for the cherry-pick yourself after
your original PR has been merged into the master branch. Once the cherry-pick PR
has merged, remove the cherry-pick label from the original PR.

For more information on contributing to the Knative Docs, see:
https://www.knative.dev/community/contributing/

 -->

Fixes #issue-number or description of the problem the PR solves

## Proposed Changes <!-- Describe the changes the PR makes. -->

-
-
-

lionelvillard commented:
this is a duplicate of this one: https://github.com/knative/docs/pull/2633

ellenevans commented:
Actually it isn't a duplicate.  CronJobSource doesn't seem to be in the
current sample set.

On Wed, Jul 8, 2020 at 2:19 PM Lionel Villard <notifications@github.com>
wrote:

> this is a duplicate of this one: #2633
> <https://github.com/knative/docs/pull/2633>
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knative/docs/pull/2660#issuecomment-655763391>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQXU27LEREIX3Y2UXJTAHTR2TPGNANCNFSM4OU6YLUA>
> .
>

lionelvillard commented:
Sorry I should have been more explicit. CronJobSource has been renamed to PingSource. I renamed it in #2633 and fixed the link.

ellenevans commented:
Thanks for the explanation.

On Thu, Jul 9, 2020 at 5:54 AM Lionel Villard <notifications@github.com>
wrote:

> Sorry I should have been more explicit. CronJobSource has been renamed to
> PingSource. I renamed it in #2633
> <https://github.com/knative/docs/pull/2633> and fixed the link.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knative/docs/pull/2660#issuecomment-656109588>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQXU27UNH2OZEGUOUW7CADR2W4XPANCNFSM4OU6YLUA>
> .
>

RichieEscarez commented:
@ellenevans should this be closed? 

if not, and this is a valid fix, I want to point out that this PR is not targeting the Master branch
(ellenevans wants to merge 1 commit into **ellenevans-patch-1** from ellenevans-patch-2)
=> To commit this change into Master, change it to:
(ellenevans wants to merge 1 commit into **Master** from ellenevans-patch-2)

ellenevans commented:
I created a branch *ellenevans-patch-2" to fix an issue I was having in
"ellenevans-patch-1", which was that a sample changed names so the link was
broken.  The theory was put -2 into -1 before I merged -1 into master.
Lionel assures me that the -2 issue is now taken care of, so -2 can go
away, and -1 should be merged.

On Fri, Jul 10, 2020 at 3:40 PM RichieEscarez <notifications@github.com>
wrote:

> @ellenevans <https://github.com/ellenevans> should this be closed?
>
> if not, and this is a valid fix, I want to point out that this PR is not
> targeting the Master branch
> (ellenevans wants to merge 1 commit into *ellenevans-patch-1* from
> ellenevans-patch-2)
> => To commit this change into Master, change it to:
> (ellenevans wants to merge 1 commit into *Master* from ellenevans-patch-2)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knative/docs/pull/2660#issuecomment-656925239>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQXU2ZCQE4IQZIZOUIB7JTR26KG7ANCNFSM4OU6YLUA>
> .
>

abrennan89 commented:
@ellenevans does this need to be merged or closed, or is it still WIP?

ellenevans commented:
Richie took care of what needed to be done.  This can go away.

Thanks,
E

On Mon, Jul 27, 2020 at 12:20 PM Ashleigh Brennan <notifications@github.com>
wrote:

> @ellenevans <https://github.com/ellenevans> does this need to be merged
> or closed, or is it still WIP?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/knative/docs/pull/2660#issuecomment-664588871>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AIQXU2YXEUG6PLOH3I43NL3R5XHOBANCNFSM4OU6YLUA>
> .
>
